makeingtables/src/Components/Posts.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import DeleteIcon from '@mui/icons-material/Delete';
+import ImportExportIcon from '@mui/icons-material/ImportExport';
+
+export default class Posts extends React.Component {
+  
+  // changeState = ()=>{
+  //   axios.get(`https://jsonplaceholder.typicode.com/${this.state.renderType}`)
+  //   .then(res => this.setState({
+  //     items:res.data.reverse()
+  //   }));
+  // }
+
+
+  render() {
+    const Posts = this.props.items;
```

Comment:
don't create variable in render , 
directly use this.props.items in place of  variable